src/views/createQuestionElement.js

```diff
@@ -1,30 +1,38 @@
 'use strict';
 
-import createDOMElement from "../utils/createDOMElement.js";
+import createDOMElement from '../utils/createDOMElement.js';
+import clickAnswerHandler from '../handlers/clickAnswerHandler.js';
 
 const createQuestionElement = (question) => {
-    const container = createDOMElement('div');
-    const title = createDOMElement('h1');
-    title.innerText = question.text;
-    container.appendChild(title);
+  const container = createDOMElement('div');
+  const title = createDOMElement('h1');
+  title.innerText = question.text;
+  container.appendChild(title);
```

Comment:
You don't have to do anything for this, just wanted to say: be careful with changing formatting in feature branches like this. It makes it look like you've changed the code here, even though you haven't. It also makes merge conflicts more likely (and much harder to resolve).

Try to never change formatting in code unless it's related to your feature. Or in a branch that you only use to fix formatting.